cmd/ui.go

```diff
@@ -78,7 +78,9 @@ func printBar(bar *pb.ProgressBar, rightText string) {
 		// TODO: check for cross platform support
 		end = "\x1b[0K\r"
 	}
-	fprintf(stdout, "%s %s%s", bar.Render(0), rightText, end)
+	rendered := bar.Render(0)
+	// Only output the left and middle part of the progress bar
+	fprintf(stdout, "%s %s %s%s", rendered.Left, rendered.Progress, rightText, end)
```

Comment:
unrelated to this PR:

Hm ... took me a while to find what `fprintf` is and where it is defined ... and the fact that it will panic if somehow we can't write to stdout seems ... bad idea to me. Do we want to continue forward with this ?

Reply:
I don't see much harm in the panic in this case, since if we can't write to stdout something is terribly wrong and panicking makes sense, instead of, say, silently running.

TBH I would get rid of `printBar()` altogether. It feels like we're abusing progress bars as simple line printers, and not for processes that require progress tracking. Especially with `ProgressBar.hijack()`, which completely, well, hijacks progress bars for this purpose.

> took me a while to find what `fprintf` is and where it is defined

You might want to setup "go to definition" in your editor/IDE. ;)